### PR TITLE
fix(FEV-861): player v7 Duplicate slide is displayed on the kitchen sink when moving from preview to live

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

filter out all slides which are duplication of their adjacent previous slides 
(happens while switching from preview to live mode on the webcast app)

solves: FEV-861

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -333,6 +333,24 @@ export const preparePendingCuepoints = (
   );
 };
 
+function filterPreviewDuplications(sortedData: Array<ItemData>) {
+  const filteredArr = [sortedData[0]];
+  for (let i = 0; i < sortedData.length - 1; i++) {
+    if (
+      !(
+        sortedData[i].title === sortedData[i + 1].title &&
```
Comment (semarche-kaltura) [inline src/utils/index.ts:341]: what if `sortedData` has length === 1,  does't the line throw an exception?

File: package-lock.json [hunk 2]
```diff
@@ -5,22 +5,17 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.10.1",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.1.tgz",
-      "integrity": "sha512-IGhtTmpjGbYzcEDOw7DcQtbQSXcG9ftmAXtWTu9V936vDye4xjjekktFAtgZsWpzTj/X01jocB46mTywm/4SZw==",
+      "version": "7.16.0",
```
Comment (semarche-kaltura) [inline package-lock.json:8]: why we have changes for `package-lock.json` in the PR? Does any dependencies should be changed?

File: src/utils/index.ts [hunk 3]
```diff
@@ -333,6 +333,24 @@ export const preparePendingCuepoints = (
...
+        sortedData[i].partnerData === sortedData[i + 1].partnerData &&
+        [sortedData[i].tags, sortedData[i + 1].tags].includes(
+          'select-a-thumb, __PREVIEW_CUEPOINT_TAG__'
```
Comment (semarche-kaltura) [inline src/utils/index.ts:344]: if cue-points have specific tag, why we can't filter cue-points only by `'select-a-thumb, __PREVIEW_CUEPOINT_TAG__'` tags without comparing `title` and `partnerData`?

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/utils/index.ts [hunk 4]
```diff
@@ -333,8 +333,13 @@ export const preparePendingCuepoints = (
...
 };
 
-function filterPreviewDuplications(sortedData: Array<ItemData>) {
-  const filteredArr = [sortedData[0]];
+function filterPreviewDuplications(
+  sortedData: Array<ItemData>
+): Array<ItemData> {
+  if (sortedData.length <= 2) {
+    return sortedData;
+  }
```
Comment (RoyBregman) [inline src/utils/index.ts:341]: Is this needed? I think it will be protected w/o it

File: package-lock.json (hunk 2 above)
Comment (JonathanTGold) [reply package-lock.json:8]: @semarche-kaltura the contrib is not updated to the newest and why not update the dependencies by the way

File: src/components/navigation/navigation-item/NavigationItem.tsx [hunk 5]
```diff
@@ -13,6 +13,8 @@ export interface RawItemData {
   title?: string;
   assetId?: string;
   subType?: itemTypes;
+  partnerData: string;
+  tags: string;
```
[the remark this answers is not in the document]
Comment (JonathanTGold) [reply src/components/navigation/navigation-item/NavigationItem.tsx:17]: yes, they are always returned with these two

**Review by RoyBregman — APPROVED** [review]
Just make sure to test it well

**Pull request merged**